client/lib/popup.js

```diff
@@ -183,7 +186,9 @@ window.Popup = new class {
       // was available and returns `false`. There is a (small) risk a false
       // positives.
       const title = TAPi18n.__(translationKey);
-      return title !== translationKey ? title : false;
+      // we need a default header [X], if we show popup as full of small screen
```

Comment:
If no title, then it's miniprofile mode. When full screen, we cannot see the header and find the [x] button to close it, so we need a default title.